### PR TITLE
Adding new spacing utilities

### PR DESCRIPTION
**Pull request by chris-pearce**

### CODE REVIEW

File: utilities/_u-spacing.scss [hunk 1]
```diff
@@ -55,6 +55,20 @@ $u-spacing-huge:    $spacing-double !default;
...
 
+// All
+%u-s-m-base,
+.u-s-m-base {@include to-rem(margin, $u-spacing-base);}
+
+%u-s-p-base,
+.u-s-p-base {@include to-rem(padding, $u-spacing-base);}
+
+// Negative
+%u-s-m-base,
```
Comment (smgcode) [inline utilities/_u-spacing.scss]: Shouldn't this be <code>u-s-m-n-base</code>?

Comment (AlecRaeside) [inline utilities/_u-spacing.scss]: yes

**Comment by AlecRaeside** [comment]
LGTM other than https://github.com/westfieldlabs/scally/pull/26/files#r17399106

File: utilities/_u-spacing.scss (hunk 1 above)
Comment (chris-pearce) [inline utilities/_u-spacing.scss]: it should, well spotted.

File: utilities/_u-spacing.scss [hunk 2]
```diff
@@ -55,6 +55,20 @@ $u-spacing-huge:    $spacing-double !default;
...
+
+%u-s-p-base,
+.u-s-p-base {@include to-rem(padding, $u-spacing-base);}
+
+// Negative
+%u-s-m-n-base,
+.u-s-m-n-base {@include to-rem(margin, -$u-spacing-base);}
+
+%u-s-p-base,
+.u-s-p-base {@include to-rem(padding, -$u-spacing-base);}
```
Comment (kllevin) [inline utilities/_u-spacing.scss]: I think these are missing the 'n'

**Comment by kllevin** [comment]
LGTM

**Pull request merged**